miscellaneous/di_i2c.py

```diff
@@ -412,6 +412,10 @@ class DI_I2C_RPI_SW(object):
         wiringpi.pinMode(8, 1) # SDA Low
         wiringpi.digitalRead(9) # SCL Read
         wiringpi.digitalRead(8) # SDA Read
+
+        # if you want to go with the BCM notation
```

Comment:
Instead of these three lines of comments, can we change the above examples to use `wiringPiSetupGpio` along with `2` and `3`?

Reply:
Done.